File: NetFramework/TouchToolkit/CS/TouchToolkitDemo/GcZoomPages/ShowControlDemo.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Drawing;
using System.Data;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using ClickOnceDemo.C1ZoomPages.Help_Forms;

namespace ClickOnceDemo.C1ZoomPages
{
    public partial class ShowControlDemo : DemoBase
    {
        public ShowControlDemo()
        {
            InitializeComponent();
            InitializeDemo();
        }

        private void InitializeDemo()
        {
            this.Title = "Show Control";
            this.Description = "When AutoShowControl property is true, C1Zoom will try to scroll the from to show the focused control.";
        }

        private void formButton_Click(object sender, EventArgs e)
        {
            using (Form form = new ShowControlForm())
            {
                form.ShowDialog(this);
                // The _C1Zoom will auto detach when the Form dispose.
                //_C1Zoom.SetEnabled(form, false);
            }
        }
    }
}

File: NetFramework/TouchToolkit/VB/TouchToolkitDemo/GcZoomPages/ShowControlDemo.vb
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Drawing
Imports System.Data
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms
Imports TouchToolKitDemo.C1ZoomPages.Help_Forms

Namespace C1ZoomPages
	Public Partial Class ShowControlDemo
		Inherits DemoBase
		Public Sub New()
			InitializeComponent()
			InitializeDemo()
		End Sub

		Private Sub InitializeDemo()
			Me.Title = "Show Control"
			Me.Description = "When AutoShowControl property is true, C1Zoom will try to scroll the from to show the focused control."
		End Sub

		Private Sub formButton_Click(sender As Object, e As EventArgs)
			Using form As Form = New ShowControlForm()
					' The _gcZoom will auto detach when the Form dispose.
					'_gcZoom.SetEnabled(form, false);
				form.ShowDialog(Me)
			End Using
		End Sub
	End Class
End Namespace
